tools/mocha-chrome-wrapper

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+const MochaChrome = require('mocha-chrome');
+const fs = require('fs');
+
+let promises = process.argv.slice(2).map(testScript => {
+  // we're not doing a ton of checking for now, since we've relied on
```

Comment:
nit: prefer capitalizing first word in comments so they read as a normal full sentence.